.percy.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+static:
+  include: '**/*.{html,htm}'
+  exclude: '**/*noImageSnapshots*.html'
+snapshot:
+  widths:
+    - 375
```

Comment:
question: Do we have stats on widths which are used? I know 414pt is typically more common nowadays than 375pt but the difference is relatively negligible and it's probably best to test the worst case.

Reply:
The iPhone X, XS (etc...) and iPhone 13 mini are all still 375, and the regular iPhone 13 is only 390, so I think this value is a sensible one.